test/k6/src/tests/instances.js

```diff
@@ -276,15 +285,134 @@ function TC10_SoftDeleteInstance(data) {
   addErrorCount(success);
 }
 
-function TC11_HardDeleteInstance(data) {
+//TC11 - Get all instances for party looked up with a person number
+function TC11_GetInstances_ByPersonNumber(data) {
+  const instanceOwnerIdentifier = `Person:${data.personNumber}`;
+  const filters = {
+    "appId": `${data.org}/${data.app}`,
+  };
+  const options = {
+    "instanceOwnerIdentifier" : instanceOwnerIdentifier,
+  };
+
+  const res = instancesApi.getInstances(data.orgToken, filters, options);
+
+  const dataBody = JSON.parse(res.body);
+  const instanciationResultSuccess = check(res, {
+    "TC11_GetInstances_ByPersonNumber: Get instance for party. Status is 200": (
+      r
+    ) => r.status === 200,
+    "TC11_GetInstances_ByPersonNumber: Instances exists for party.": 
+    dataBody.count > 0,
+  });
+  addErrorCount(instanciationResultSuccess);
+
+  if (dataBody.count > 0) { // If instance(s) exists
+    const firstInstance = dataBody.instances[0];
+    const personNumber = firstInstance.instanceOwner.personNumber;
+    const instanceIdSplit = firstInstance["id"].split("/");
+
+    const success = check(res, {
+      "TC11_GetInstances_ByPersonNumber: Get instance for party. InstanceId has expected format":
+        instanceIdSplit.length === 2,
```

Comment:
once we get to this test we are already confident the format is as expected as this is covered by an earlier TC.  Can limit the number of checks to what is relevant for the feature being tested.